pkg/cri/store/stats/stats.go

```diff
@@ -1,27 +1,71 @@
-/*
-   Copyright The containerd Authors.
+package stats
 
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
+import (
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
 
-       http://www.apache.org/licenses/LICENSE-2.0
+// ContainerStats contains the information about container stats.
+type ContainerStats struct {
+	Attributes      *ContainerAttributes
+	CPUStats        *ContainerCpuStats
+	MemoryStats     *ContainerMemoryStats
+	FileSystemStats *ContainerFileSystemStats
+	CPUStatsUpdate  *ContainerCpuStatsUpdate
```

Comment:
I would suggest to maybe not make these pointers, and instead embedded the struct. This way we will get the zero values if they are not filled in.